content/sites/ignore-assets.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Ignoring Subsets of Static Assets
+alwaysopen: true
+weight: 5
+---
+
+Worker Sites require [Wrangler](https://github.com/cloudflare/wrangler) - make sure to be on the [latest version](/quickstart/#updating-the-cli) - and the Workers [Unlimited plan](https://workers.cloudflare.com/sites#plans).
+
+There are cases where users may not want to upload certain static assets to their Workers Sites.
+In this case, Workers Sites can also be configured to ignore certain files or directories using logic 
```

Comment:
Is there a reason to have line breaks after each of these sentences?

Reply:
Just makes it more readable in text editors; markdown will format without these newlines.

Reply:
cool